app/_hub/stone-payments/kong-plugin-template-transformer/index.md

```diff
@@ -0,0 +1,53 @@
+---
+name: Template Transformer
+publisher: stone-payments
+categories:
+  - transformations
+type: plugin
+desc: Kong middleware to transform requests / responses, using pre-configured templates.
+description: |
+    This is Kong plugins that accepts requests and response templates to completely transform requests and responses with Lua templates.
+support_url: https://github.com/stone-payments/kong-plugin-template-transformer/issues
+source_url:  https://github.com/stone-payments/kong-plugin-template-transformer
+license_type: Apache-2.0
+license_url: https://github.com/stone-payments/kong-plugin-template-transformer/blob/master/LICENSE
+
+kong_version_compatibility:
+  community_edition:
+    compatible:
+     - 0.13.x
+     - 0.14.x
+    incompatible:
```

Comment:
Note that Kong Enterprise versions 0.32 and 0.33 are build on Kong version 0.13.x https://docs.konghq.com/enterprise/changelog/#032---20180524

Perhaps you could add compatibility with Kong Enterprise 0.32 and 0.33 here? Or are you fairly certain that this plugin is NOT compatible with Kong Enterprise?